### PR TITLE
Meeting Plugin - Added Recording Events 'started', 'stopped', 'paused', 'resumed' 

### PR DESCRIPTION
**Pull request by akoushke**
Fixes #[INSERT LINK TO ISSUE NUMBER]
https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-165740
---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/node_modules/@webex/plugin-meetings/src/constants.js [hunk 1]
```diff
@@ -381,6 +388,10 @@ export const EVENT_TRIGGERS = {
   MEETING_STOPPED_SHARING_LOCAL: 'meeting:stoppedSharingLocal',
   MEETING_MEDIA_LOCAL_STARTED: 'meeting:media:local:start',
   MEETING_MEDIA_REMOTE_STARTED: 'meeting:media:remote:start',
+  MEETING_STARTED_RECORDING: 'meeting:startedRecording',
+  MEETING_STOPPED_RECORDING: 'meeting:stoppedRecording',
+  MEETING_PAUSED_RECORDING: 'meeting:pausedRecording',
+  MEETING_RESUMED_RECORDING: 'meeting:resumedRecording',
```
Comment (mwegman) [inline packages/node_modules/@webex/plugin-meetings/src/constants.js:394]: You could go with `meeting:recording:<verb>`

File: packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js [hunk 2]
```diff
@@ -61,6 +62,7 @@ export default class LocusInfo extends EventsScope {
     this.membership = locus.membership || null;
     this.identities = locus.identities || null;
     this.participants = locus.participants || null;
+    this.contorls = locus.controls || null;
```
Comment (arun3528) [inline packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js:65]: you will need to remove this line as , It wont trigger the initial event when someone joins

File: packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js [hunk 3]
```diff
@@ -635,8 +637,34 @@ export default class LocusInfo extends EventsScope {
...
+        let state = null;
+
+        if (updates.hasRecordingChanged) {
+          state = current.record.recording ? RECORDING_STATE.RECORDING : RECORDING_STATE.IDLE;
+        }
+        else if (updates.hasRecordingPausedChanged) {
+          state = current.record.paused ? RECORDING_STATE.PAUSED : RECORDING_STATE.RESUMED;
+        }
+
+        this.emitScoped(
```
Comment (arun3528) [inline packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js:653]: you only need to update when there is a change !

File: packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js (hunk 2 above)
Comment (akoushke) [reply packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js]: removed

**Pull request merged**